.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
```

Comment:
- `required`
- **Well done so far👏👏**

- [ ] Please ensure to summarize and gives an overall idea about the review in the review body ✨.

- [ ] Kindly Highlight some of the good work that the student was able to achieve in this milestone 👏👏👏

- [ ] I suggest you add emoji's to your comments so that the student understands the tone you are trying to communicate 😎.

- [ ] Please explain the **why** you are asking for that change by providing examples, like how to fix linters or provide some of the best practices approach ✔️  so the student can learn something 👨‍🏫😄 as well and don't feel the change was unfair 👌.